### PR TITLE
migrate aws arm/amd profiles to multiarch profiles on 4.12/4.13/4.14

### PR DESCRIPTION
**Pull request by lwan-wanglin**
refer https://issues.redhat.com/browse/OCPQE-18164

- migrate aws arm/amd profiles to multiarch profiles on 4.12/4.13/4.14

related 4.15 aws migration pr: https://github.com/openshift/release/pull/46909

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
@yunjiang29 could you help to review? thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 1]
```diff
@@ -84,6 +169,274 @@ tests:
...
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @disconnected'
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-imdsv2-arm-f28
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:325]: cc @sunzhaohua2 for awareness

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 2]
```diff
@@ -84,6 +169,274 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi'
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-imdsv2
+- as: aws-ipi-ingress-lbtype-classic-arm-f28
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:353]: cc @lihongan for awareness, including all other ingress profiles

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 3]
```diff
@@ -84,6 +169,274 @@ tests:
...
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and not @fips'
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-ovn-efs-arm-f28
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:407]: cc @duanwei33 for awareness, including all other storage related profiles

**Comment by yunjiang29** [comment]
@liangxia @jhou1 can you approve this PR? thanks

**Comment by yunjiang29** [comment]
cc @jianlinliu

**Comment by liangxia** [comment]
There are conflicts.

**Comment by lwan-wanglin** [comment]
@liangxia no conflicts now, PTAL again, thanks

**Pull request merged**